app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  include Response
+
+  def rent_charges_use_case_factory
```

Comment:
Looks like this method could be reusable in `view historical percent changes` controller too. So, I think it is better to use a generic name that can be used by other controllers.

Reply:
sure, I've just renamed it to `use_case_factory` for now